Packs/CrowdStrikeMalquery/ReleaseNotes/1_0_1.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### CrowdStrike Malquery
+**Breaking Change:** **file** command changed to return multiple entries (entry per indicator) instead of a single entry.
```

Comment:
```suggestion
**Breaking Change:** The ***file*** command was changed to return multiple entries (entry per indicator) instead of a single entry.
```